generators/client/templates/vue/src/main/webapp/swagger-ui/index.html.ejs

```diff
@@ -98,6 +98,10 @@
                     if (authToken) {
                         req.headers['Authorization'] = "Bearer " + authToken;
                     }
+                    // Remove the sample Swagger UI request body if present
+                    if (req.body === '{"additionalProp1":"string","additionalProp2":"string","additionalProp3":"string"}') {
```

Comment:
same, you need to add this part above too, for oauth2 configuration

Reply:
I have added it for other auth methods as well. Good catch @pascalgrimaud !